### PR TITLE
localization in todo app

### PR DESCRIPTION
**Pull request by kirosamir29**

### CODE REVIEW

File: assets/lang/ar.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "change_lang": "English",
```
Comment (MinaSamirCap) [inline assets/lang/ar.json]: change all name as its

File: lib/screens/add_todo_screen.dart [hunk 2]
```diff
@@ -36,25 +37,25 @@ class _AddTodoScreenState extends State<AddTodoScreen> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: Text(isEdit ? "Edit Todo" : "Add Todo"),
+        title: Text(isEdit ? "edit_title".tr(context) : "add_title".tr(context)),
```
Comment (MinaSamirCap) [inline lib/screens/add_todo_screen.dart]: Change to const

**Review by MinaSamirCap — CHANGES_REQUESTED** [review]

File: assets/lang/ar.json (hunk 1 above)
Comment (kirosamir29) [reply assets/lang/ar.json]: done

File: lib/screens/add_todo_screen.dart (hunk 2 above)
Comment (kirosamir29) [reply lib/screens/add_todo_screen.dart]: done

**Pull request merged**